invenio_rdm_records/theme/templates/invenio_rdm_records/details/doi.html

```diff
@@ -0,0 +1,35 @@
+{#
+  Copyright (C) 2020 CERN.
+  Copyright (C) 2020 Northwestern University.
+
+  Invenio RDM Records is free software; you can redistribute it and/or modify
+  it under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- set id_doi = record.identifiers|doi_identifier %}
+<dt>{{_('DOI')}}:</dt>
+<dd>
+  <span class="get-badge" data-toggle="tooltip" data-placement="bottom"
```

Comment:
minor: I would never have known this was possible without looking at the code! Neat. Could we add css to make cursor a pointer: `cursor: pointer;` when hovering over the badge?

Reply:
Yup, done :)